Documentation/devel/release.md

```diff
@@ -1,9 +1,22 @@
 # rkt release guide
 
-How to perform a release of rkt.
-This guide is probably unnecessarily verbose, so improvements welcomed.
-Only parts of the procedure are automated; this is somewhat intentional (manual steps for sanity checking) but it can probably be further scripted, please help.
+## Release cycle
+
+This section describes the typical release cycle of rkt:
+
+1. A [milestone](https://github.com/coreos/rkt/milestones) is created on GitHub, fixing the target date for a future release (typically every 2/3 weeks).
+2. Tickets milestoned for the next release are worked on, accordingly to their priorities.
+3. PRs are submitted in order to introduce changes. Those needs to be reviewed and tests must not be failing in order to be accepted.
+4. The day before the release there is a short freeze period, with a moratorium on changes. No new code nor dependencies can be merged on this day, which focuses instead on soft activities:
+  * documentation improvements and usability tests
+  * roadmap planning and scheduling of next milestones
+  * triaging and backlog sorting
+  * building, testing and preparing the release (by a release manager)
 
+## Release process
+
+This section shows the whole process involved in performing a release of rkt. It is probably unnecessarily verbose, so improvements welcomed.
```

Comment:
"This section shows how to perform a release of rkt."

(Delete second sentence on this line.)